lab 2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>My title</title>
+
+      <!-- font awesome -->
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+   </head>
+   <body>
+   <header>
+      <div class="collapsed-menu">
+         <div class="title menu-item">Navigation</div>
+         <div class="button menu-item"><i class="fa fa-bars" aria-hidden="true"></i></div>
+      </div>
+      <div class="menu">
+         <div class="menu-item"><a href='#'>psd websites</a></div>
+         <div class="menu-item"><a href='#'>ui kits</a></div>
+         <div class="menu-item"><a href='#'>premium psd files</a></div>
+         <div class="menu-item"><a href='#'>about</a></div>
+         <div class="menu-item"><a href='#'>blog</a></div>
+      </div>
+   </header>
+   <article>
+      <div class="page page-1">
+         <section>
+            <div class="content-main">
+               <h1>Awesome Free PSD <br> templates &amp; Resources</h1>   
+            </div>
+            <a href="#">
+               <div class="link" style="display: flex; justify-content: space-between; align-items: center;">
+                  <span>www.BlazRobar.com</span>
+                  <i class="fas fa-1x fa-angle-right"></i>
+               </div>
+            </a>
+         </section>
+      </div>
+      <div class="page page-2">
+         <div class="line first-line">
+            <div class="text">
+               <p>I'm like the Robin Hood of PSD files.</p>
+            </div>
+            <div class="arrow">
+               <i class="fas fa-1x fa-angle-right"></i>
+            </div>
+         </div>
+         <div class="line second-line">
+            <div class="arrow">
+               <i class="fas fa-1x fa-angle-right"></i>
+            </div>
+            <img src="./img/bubble.jpg">
+            <img src="./img/criceto.jpg">
+         </div>
+         <div class="third-line">
+            <div class="title"><p>BlazRobar.com</p></div>
```

Comment:
if it's title, than it's better to use h1 h2 h3 h4 ...

Reply:
#done